app/controllers/organizations_controller.rb

```diff
@@ -109,4 +112,17 @@ def valid_filename?
     @organization.errors.add(:profile_image, FILENAME_TOO_LONG_MESSAGE)
     false
   end
+
+  def rate_limit!
+    rate_limiter.tap do |rate_limiter|
```

Comment:
I don't think we need `tap` here, as we're ignoring the return value of this method anyway.

Reply:
I was following the convention @rhymes setup, as he stated, this will get refactored so I Im going to leave it for now.